tensornetwork/backends/numpy/numpy_backend_test.py

```diff
@@ -0,0 +1,106 @@
+"""Tests for graphmode_tensornetwork."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import numpy as np
+import tensornetwork
+from unittest import TestCase
+from tensornetwork import set_default_backend
+import tensorflow as tf
+
+
+class BasicNumPyBackendTest(TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    set_default_backend('numpy')
+
+  def test_tensordot(self):
+    net = tensornetwork.TensorNetwork()
+    a = net.add_node(2*np.ones((2, 3, 4)))
+    b = net.add_node(np.ones((2, 3, 4)))
+    tensordotted_tensor = net.backend.tensordot(a.tensor, b.tensor, ((1, 2), (1, 2)))
+    self.assertListEqual(np.array(tensordotted_tensor).tolist(), [[24.0, 24.0], [24.0, 24.0]])
```

Comment:
We prefer to use `pytest` for our testing. Can you remove your dependency on unittest and use pytest instead?

Reply:
Sure, was not using correct line-length anyway, so need to revisit all the code : )

Reply:
Yes we should. That was likely left over from the initial open source release